Documentation/design/operatorgroups.md

```diff
@@ -0,0 +1,221 @@
+# Operator Multitenancy with OperatorGroups
+
+An `OperatorGroup` is an OLM resource that provides rudimentary multitenant configuration to OLM installed operators.
+
+## OperatorGroup Overview
+
+* An `OperatorGroup` selects a set of target namespaces in which to generate required RBAC access for its member operators.
+* The set of target namespaces is provided via a comma-delimited string stored in the `olm.targetNamespaces` annotation. This annotation is applied to member operator's deployments and is accessible through [The Downward API](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api)
+* An operator is said to be a [member of an `OperatorGroup`](#operatorgroup-membership) if its CSV exists in the same namespace as the `OperatorGroup` and its CSV's [`InstallModes` support the set of namespaces targetted by the `OperatorGroup`](#installmodes-and-supported-operatorgroups)
```

Comment:
Besides being in the same namespace, `InstallModes` are the primary factor in determining membership. I think it makes sense for it to be in the overview. `InstallModes` are tightly coupled to membership so I feel that putting it in a separate bullet will dilute that association. 
The link serves as a pointer to the definition since I don't want to leave people wondering or `ctrl-f`ing.